### ISSUE TITLE
[SEO] Enable a way to reference meta for pages of a site

### ISSUE THREAD

**Issue by srenault-meeds** [Meeds-io/meeds#3110]
## Summary  

Effective Search Engine Optimization (SEO) is crucial for enhancing the visibility and accessibility of web content. Currently, **the Meeds Builders App** lacks the capability for administrators to manage essential SEO elements on a per-page basis, specifically:
- <title> Tag: The inability to define custom titles for individual pages limits control over how pages are represented in search engine results.
- <meta> and <link> Tags: The absence of functionality to inject custom HTML into the <head> section of pages restricts the ability to add meta descriptions, canonical links, Open Graph tags, and alternate links, all of which are vital for SEO.

Addressing these limitations is essential to improve SEO performance of **the Meeds Builders App**  and the public site.

## Problem Statement  
We cannot control what is referenced, and most of all, we cannot tag pages so it is well optimized.

## Proposed Solution  
Enable to reference meta infos for pages of any site in the **builders extension only**
No UI is required at all/

This referencing would be done by customizing infos in the Builders Extension's code.

Following requirements are expected:
- a non-dev (certified to push code of course) can edit and can provide then PR for SEO optimization
 
- this user can list any page of the builders and for each page, it is possible to add meta tags as following items for example:
   - custom title to replace the one dynamically inheritated from the page name
   - meta name = description
   - link rel properties like href
   - meta properties like og:image, og:image:alt, og:image: width...
   
- these infos must be added at the start of head

- No multilingual is required for now

## Benefits  
The Builders public site is referenced when searching "Community engagement platform" using the engine like google (if of course this info is added within the tags.
